__test__/pages/account/account.test.tsx

```diff
@@ -111,8 +111,13 @@ describe("MyAccount page", () => {
     expect(() =>
       render(
         <MyAccount
-          pickupLocations={null}
-          patron={processedPatron}
+          accountData={{
+            pickupLocations: filteredPickupLocations,
+            checkouts: null,
```

Comment:
You shouldn't need to pass null values explicitly if these attributes are optional.